test/restart.jl

```diff
@@ -27,6 +29,12 @@ ClimaComms.init(comms_ctx)
 # 2. A simulation, saved to a previous checkpoint, and read back and evolved to
 #   the same time is identical (up to some tolerance)
 # 3. ClimaAtmos can automatically detect restarts
+#
+# This test checks a variety of configurations and spends a long time compiling
+# functions. For this reason, the test takes an optional command-line argument
+# to only many tests instead of comprehensive test.
+
+const MANYTESTS = length(ARGS) >= 1
```

Comment:
Can we make and extract this as a kwarg instead?

Reply:
Would this be fine:
```julia
const MANYTESTS = "--manytest" in ARGS
```
?

Reply:
Yeah, I think that's fine